msamp/deepspeed/runtime/engine.py

```diff
@@ -4,24 +4,20 @@
 # SPDX-License-Identifier: Apache-2.0. DeepSpeed Team)
 
 """DeepSpeedEngine in MS-AMP."""
-
```

Comment:
It should be another PR?

Reply:
The change of this file is mainly for supporting "msamp" section in deepspeed config. It is also for this example. I know it will make this PR larger and hard to review, but I think it is acceptable since they are related and break to multiple PRs will take longer time for this example.